packages/playground/website/src/main.tsx

```diff
@@ -1,22 +1,31 @@
 import { createRoot } from 'react-dom/client';
-
 import PlaygroundViewport from './components/playground-viewport';
 import ExportButton from './components/export-button';
 import ImportButton from './components/import-button';
+import VersionSelector from './components/version-select';
+import './styles.css';
 
 import { setupPlayground } from './lib/setup-playground';
 
+const phpVersions = ['8.2', '8.0', '7.4', '7.3', '7.2'];
```

Comment:
Also 7.1, 7.0, 5.6

Reply:
Would be cool to source these versions from a common source of truth, I need to think about it more. Not a blocker here